package.json

```
@@ -32,11 +34,12 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "gatsby build",
+    "build": "npm run lint && gatsby build",
```

I'm a bit unsure about this, especially when it comes to our CI.

I would much rather have two steps in our CI Action, one for linting and one for build if necessary. The build will still fail but it'll be easier to see exactly what failed, the build or lint step.

Take a look at the suggested changes to the ci.yml above. @eluciano11 thoughts?

```suggestion
    "build": "gatsby build",
```

---

Agreed!